config/locales/en.yml

```diff
@@ -322,3 +323,9 @@ en:
         manager: Manager
         developer: Developer
         guest: Guest
+
+  activity:
+    actions:
+      update: 'updated'
```

Comment:
the key is `update` and the value is `updated`. What do you think of renaming it to `updated`?